app/models/user.js

```diff
@@ -25,5 +26,12 @@ module.exports = (sequelize, DataTypes) => {
       }
     }
   );
+
+  User.prototype.getToken = () => {
+    const { id, email } = this;
+
+    return jwtHelper.getToken({ id, email });
```

Comment:
I prefer to use this function (jwtHelper.getToken) directly in the signIn service.